zerver/lib/webhooks/git.py

```diff
@@ -247,7 +248,7 @@ def get_issue_event_message(
         user_name,
         action,
         url,
-        number,
+        number=number,
```

Comment:
Can you use mandatory kwargs in `get_issue_event_message` too?

Reply:
Sure, should I make a seperate commit to this PR or create a new PR about it?

Reply:
I'd do it in a separate commit in this PR.

You can also squash the two existing commits in this PR into each other -- it's logically one change to move to required kwargs and adjust all callers to pass them.